src/main/java/org/opentripplanner/api/model/ApiRouterInfo.java

```diff
@@ -31,7 +32,7 @@
     
     @JsonSerialize(using= GeometrySerializer.class)
     @JsonDeserialize(using= GeometryDeserializer.class)
-    @XmlJavaTypeAdapter(value=GeometryAdapter.class,type=Geometry.class)
+    @XmlJavaTypeAdapter(value= GeometryAdapter.class,type=Geometry.class)
```

Comment:
Eliminate use of adapter here

Reply:
After looking closer at this it is a bit of work to remove the Adapters for the external types, so I left them as is. The XML adapters are removed, with the removal of XML support.